public/locales/en.json

```diff
@@ -389,11 +389,31 @@
             },
             "preferences": {
                 "title": "Preferences",
-                "theme": "Editor theme",
-                "keyMap": "Keymap",
-                "indentWithTabs": "Tab character",
+                "theme": {
+                    "label": "Editor theme",
+                    "one-dark": "Dark",
+                    "neat": "Light"
+                },
+                "keyMap": {
+                    "label": "Keymap",
+                    "sublime": "Sublime",
+                    "emacs": "Emacs",
+                    "vim": "Vim"
+                },
+                "indentWithTabs": {
+                    "label": "Tab character",
+                    "on": "Tabs",
+                    "off": "Spaces"
+                },
                 "indentUnit": "Tab size (when using spaces)",
-                "spellcheck": "Spell checking"
+                "spellcheck": {
+                    "label": "Spell checking"
+                },
+                "ligatures": {
+                    "label": "Show ligatures",
+                    "on": "On",
+                    "off": "Off"
```

Comment:
Aren't these common terms that should be placed under the `common.`-namespace?

Reply:
The words "on" and "off" yes. But in this case they are used in a special context. We could also write "Yoink" and "Yeet".

Also I don't want to add an exception to the code.

Reply:
Yes, but we should minimize the required translation maintenance operations. When we have multiple on/off, yes/no, etc. translations, we could come to the point that these terms are not the same through the application. And if we want to manually maintain those cases, we'd have a lot of extra work.

Therefore I ask that you either add an exception to your code or rename these to "Ligatures enabled/disabled" or similar.

Reply:
I chose the exception. But not now. I already did it when I implemented the toggle :sweat_smile: . I forgot that the ligature toggle is a custom component and not a boolean property component.